tests/samarium/test_run.py

```diff
@@ -0,0 +1,18 @@
+from tests import Samarium
+
+
+def test_basics():
+    with Samarium(file="tests/samarium/hello_world.sm") as s:
```

Comment:
Wouldn't it be less expensive to use parametrization and fixtures...?

Reply:
You can't exactly specify a file for a fixture